.pipelines/build.yml

```diff
@@ -0,0 +1,653 @@
+# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
+#       env:
+#           MY_ACCESS_TOKEN: $(System.AccessToken)
+# Variable 'MajorVersion' was defined in the Variables tab
+# Variable 'MinorVersion' was defined in the Variables tab
+trigger:
+  branches:
+    include:
+    - refs/heads/master
+  batch: True
+schedules:
+- cron: 0 18 * * *
+  branches:
+    include:
+    - refs/heads/master
+name: $(MajorVersion).$(MinorVersion).$(date:yyMMdd)$(rev:.r)
+jobs:
+- job: BuildBinaries
+  pool:
+    name: Azure Pipelines
+    vmImage: 'windows-2022'
+    demands:
+    - msbuild
+  strategy:
+    matrix:
+      x86:
+        buildPlatform: 'x86'
+      x64:
+        buildPlatform: 'x64'
+      arm:
+        buildPlatform: 'arm'
+      arm64:
+        buildPlatform: 'arm64'
+  steps:
+  - checkout: self
+    clean: true
+    persistCredentials: True
+
+  - task: NuGetToolInstaller@1
+    displayName: Use NuGet 6.0.2
+    continueOnError: True
+    inputs:
+      versionSpec: 6.0.2
+  - task: NuGetCommand@2
+
```

Comment:
The line breaks here are strange.  There should be a blank line between tasks.  There should not be a line break between a task and its name.